packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -80,7 +86,7 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
         const rowHeight = parseFloat(controlHeight.getValueFor(this.table))
             + 2 * parseFloat(borderWidth.getValueFor(this.table));
         return {
-            count: this.table.tableData.length,
+            count: this.tanStackTable.getRowModel().rows.length,
```

Comment:
No, since `getRowModel()` is not a member on `this.table` (which refers to the Nimble `Table`. The `Table` has no APIs that would reveal how many rows are ultimately visualized. Only the `TanStackTable` knows this. I don't think we want to make the `TanStackTable` a part of the public API of the Nimble `Table`. I mean, we can mark it `internal`, or even leverage the new `ElementInternals` API we recently discussed (maybe?), but I think we can save that for a later refactor if we wish.

Reply:
Another option to consider is to pass the number of rows to the virtualizer so the virtualizer doesn't have to determine it.

Reply:
I'd rather mark in internal if we are already sharing the state if that's okay and accessing it with `this.table.table`

Reply:
> Another option to consider is to pass the number of rows to the virtualizer so the virtualizer doesn't have to determine it.

oh true, we might need a way to do it dynamically though? Maybe we can make an internal method on the table `getRowCount()`?

Reply:
I don't think the virtualizer ever triggers an update on itself, so the need for that information isn't extremely dynamic. It looks like the count would have to be passed to `connectedCallback()` and `dataChanged()`. I don't have an issue exposing an internal `getRowCount()` from the table either though.